hydra-cardano-api/hydra-cardano-api.cabal

```diff
@@ -144,6 +146,7 @@ library
     , ouroboros-network               ==0.1.0.1
     , plutus-ledger-api               ==1.0.0.0
     , QuickCheck
+    , relude
```

Comment:
This will hinder upstreaming the whole `hydra-cardano-api`

What specifically were you missing?

Reply:
Main thing is more general `show` function. That is why I introduced relude in the first place. I have fixed the return types to `String` to keep the dependencies at minimum.